google/cloud/bigtable/examples/bigtable_examples_common.cc

```diff
@@ -171,10 +172,8 @@ google::cloud::bigtable::examples::Commands::value_type MakeCommandEntry(
         argv.size() != 3 + args.size()) {
       std::ostringstream os;
       os << name << " <project-id> <instance-id> <table-id>";
-      char const* sep = " ";
-      for (auto const& a : args) {
-        os << sep << a;
-      }
+      if (!args.empty()) os << " ";
+      os << absl::StrJoin(args, " ");
```

Comment:
Looks good to me.

Nit: For my money, I'd coalesce the ones that now test `empty()`.  For example,
```
  if (!args.empty()) os << ' ' << absl::StrJoin(args, ' ');
```

Reply:
Agreed.